addons/purchase/views/purchase_views.xml

```diff
@@ -652,4 +656,179 @@
                 </search>
             </field>
         </record>
+
+    <!-- Dashboard action buttons -->
+    <!-- Unforunately to keep consistent view we have to duplicate actions for every view -->
```

Comment:
To avoid adding lot of actions, it is maybe better to create then on fly in the JavaScript (in the function _onDashboardActionClicked) and passed the data needed (only the search filter by default, if I understand well) in the html (with data-search-filter="draft_rfqs,my_purchases") ? What do you think ?

Reply:
Thanks for you help on this! It's now reduced to 2 actions with the context passed from the custom dashboard template's attributes.